Control Portal/using sAML for single-sign-on to the centurylink platform control portal with windows 2012R2.md

```diff
@@ -0,0 +1,225 @@
+
+SAML has three main parties: the user, the identity provider (IdP), and service provider (SP). The IdP is the repository that holds identity information. The SP is the party that wants to authenticate a particular user who is using an application.
+
+The SAML flow occurs as shown below.
+![SAML Flow](../images/saml01.png)
+
+Specific steps in this flow are:
+
+1. The enterprise user of the CenturyLink Cloud hits a URL that is dedicated to their account. The user is asked how they would like to log into the system and they choose SAML.
+
+2. The web application contacts the CenturyLink Cloud SAML service to initiate the SAML message exchange.
+
+3. The CenturyLink Cloud SP sends a digitally signed SAML authentication request to the enterprise IdP. This IdP takes the user's Kerberos token and validates them as a user on the enterprise network.
+
+4. The IdP returns a signed (and optionally, encrypted) SAML authentication response message to the CenturyLink Cloud SP. This message includes a Name ID assertion and that value is matched to a User record in the CenturyLink Cloud.
+
+5. The user is logged into the CenturyLink Cloud and operates under the roles and permissions assigned to their CenturyLink Cloud user account.
+
+The steps below walk through the process of building an entire SSO and SAML scenario based on Microsoft Active Directory Federation Services as the IdP proxy. If you already have an identity provider, you can skip to step 3 where trust is established between CenturyLink Cloud and the IdP.
+
+###Prerequisites
+All steps in this guide require Domain Administrator permissions in Active Directory.  This guide assumes you have the Account Administrator Role in the CenturyLink Cloud Control Panel.  This guide will use a valid public domain name and valid public DNS for name resolution needs.  If you do not have access to a valid public domain name there will be an Additional Information section at the end provided that will enable you to test this scenario, without using valid public domain names using hosts file entries in your operating system.  This guide will use private SSL certificates that are signed by the Certificate Authority that is created while following this guide.  Some organizations will prefer the use of third party trusted certificates.  This guide is still applicable except you will use your already requested and signed certificates from your trusted third party source.
+
+**Steps**
+
+**1. Provision server to act as Identity Provider.**
+
+Log into the CenturyLink Cloud Control Portal and choose to create a new Blueprint. This takes you to the Blueprint Designer wizard.
+
+![Blueprint Menu](../images/ADFS SAML 02.png)
```

Comment:
Can we use a screenshot that isn't full screen on a large browser. It looks weird because we center the big green monster text but you only see part of it so it looks like the menu is broken,